src/Psalm/Internal/Diff/ClassStatementsDiffer.php

```diff
@@ -109,13 +109,13 @@ function (
                 }
 
                 if ($a instanceof PhpParser\Node\Stmt\ClassMethod && $b instanceof PhpParser\Node\Stmt\ClassMethod) {
-                    if ((string) $a->name !== (string) $b->name) {
+                    if ((string)$a->name !== (string)$b->name) {
```

Comment:
These whitespace changes seem unrelated.

Reply:
You're right, they must have slipped through. I reverted them